src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -880,6 +880,27 @@ interface i1
                 Diagnostic(ErrorCode.ERR_EventPropertyInInterface, "add"));
         }
 
+        [Fact]
+        public void CS0065ERR_EventNeedsBothAccessors_Interface05()
+        {
+            var text = @"
+public interface I2 { }
+
+public interface I1
+{
+    abstract event System.Action I2.P10;
+}
+";
+            CreateCompilation(text).VerifyDiagnostics(
+                Diagnostic(ErrorCode.ERR_ExplicitEventFieldImpl, ".").WithLocation(6, 36),
```

Comment:
>Diagnostic(ErrorCode.ERR_ExplicitEventFieldImpl, ".").WithLocation(6, 36), [](start = 16, length = 74)

Please include comments with the error message and the corresponding source line (similar to the test below). If you change the code to ```CreateCompilation(text).VerifyDiagnostics();``` and run the unit-test, its output will have all that ready to copy/paste in the "Actual" section.